examples/cstg/html/index.html

```diff
@@ -6,190 +6,118 @@
     <link rel="stylesheet" type="text/css" href="/stylesheets/app.css" />
     <link rel="shortcut icon" href="/images/favicon.png" />
     <script defer src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
-    <script defer src="{{ UID2_JS_SDK_URL }}"></script>
-    <script defer src="{{ EUID_JS_SDK_URL }}"></script>
+    <script defer src="<!--# echo var='SSI_UID_JS_SDK_URL' -->"></script>
     <script>
       const clientSideIdentityOptions = {
-        subscriptionId: '{{ SUBSCRIPTION_ID }}',
-        serverPublicKey: '{{ SERVER_PUBLIC_KEY }}',
+        subscriptionId: '<!--# echo var="SSI_SUBSCRIPTION_ID" -->',
```

Comment:
I'm curious about the switch to SSI - the syntax isn't as nice and it doesn't seem like we need any of the SSI features.